packages/neo-one-client-core/src/errors.ts

```diff
@@ -3,6 +3,10 @@ import { makeErrorWithCode } from '@neo-one/utils';
 import BigNumber from 'bignumber.js';
 
 export const UnknownNetworkError = makeErrorWithCode('UNKNOWN_NETWORK', (name: string) => `Unknown network ${name}`);
+export const UnknownAddressError = makeErrorWithCode(
```

Comment:
We probably don't need both `UnknownAddressError` and `UnknownAccountError` - let's just stick with `UnknownAccountError`